include/bn_utils.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Remove all copyrights year. Even better, add the short version instead.
```c
/**
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0.
 */
```

Reply:
You can just update the date for now and replace all copyrights with the smaller one in a separate PR.

Reply:
I've gone ahead and used the smaller copyright for all the new files I added! Then the rest can be done in a separate PR.